twine/commands/upload.py

```diff
@@ -85,7 +86,11 @@ def upload(upload_settings, dists):
         elif upload_settings.sign:
             package.sign(upload_settings.sign_with, upload_settings.identity)
 
-        resp = repository.upload(package)
+        # Suppress TLS verification warning on trusted custom certs
+        # See https://docs.python.org/3/library/warnings.html#temporarily-suppressing-warnings
```

Comment:
```suggestion
```

Kudos for the inclination to provide a helpful link, but I think the code is clear enough, and hopefully it's easy enough for folks to find the relevant docs.

The alternative is adding `# noqa: E501` to the end of the long line, but that feels noisy to me.

Reply:
Hi, just fixed these issues as suggested and pushed the fork. Sorry to be so late.